internal/controllers/clusterextension_controller_test.go

```diff
@@ -561,28 +495,6 @@ func TestClusterExtensionInstallationSucceeds(t *testing.T) {
 	require.NoError(t, cl.DeleteAllOf(ctx, &ocv1alpha1.ClusterExtension{}))
 }
 
-func verifyInvariants(ctx context.Context, t *testing.T, c client.Client, ext *ocv1alpha1.ClusterExtension) {
```

Comment:
Interesting that the first test is the only place this is being called. So much for invariants if we only test them in one scenario. Haha.

I think we've decided to loosen our invariant checks, but want to call this out to make sure we talk about it.

Reply:
The lint pointed out that , these functions are not used anymore.